src/Router.php

```diff
@@ -17,14 +17,71 @@ public static function getInstance()
         return static::$instance;
     }
     
+    /**
+     * Loads the location of the routes file.
+     * 
+     * @return string
+     */
+    protected function getRouteFileLocation()
+    {
+        $dir = getenv('HOME') . '/.volcano-sdk';
+        
+        if (!is_dir($dir)) {
+            mkdir($dir);
+        }
+        
+        return $dir . '/routes.json';
+    }
+    
+    /**
+     * Loads Routes
+     */
     public function loadRoutes()
     {
+        $rfile = $this->getRouteFileLocation();
+        
+        if (!file_exists($rfile)) {
```

Comment:
@cthos Currently. the cached routes file is saved indefinitely, right? Do you think the SDK should automatically recache the routes, say, once a month - for easy upgrades?

For example, we could alter the output of the Endpoints API to include a `built_at` timestamp. Then we could force load `$this->loadRoutesFromService()` if the cached routes file doesn't exist or if its `built_at` is older than 30 days.